pkg/repro/repro.go

```diff
@@ -272,7 +279,6 @@ func (ctx *context) extractProgSingle(entries []*prog.LogEntry, duration time.Du
 		UseTmpDir:  true,
 		HandleSegv: true,
```

Comment:
Please move this to a separate function, a-la createDefaultOps, and reuse it in extractProgBisect. Easy to forget to set/unset an option in both places.